frontend/types/types.ts

```diff
@@ -358,13 +358,26 @@ export enum MarketPlaceModelGroup {
 }
 
 export type NewModelData = {
-  teamName: string
-  modelName: string
+  name: string
   description: string
   visibility: 'public' | 'private'
 }
 
-export type Artefact = any
+export const ModelVisibility = {
+  Private: 'private',
+  Public: 'public',
+} as const
+
+export type ModelVisibilityKeys = (typeof ModelVisibility)[keyof typeof ModelVisibility]
+
+export interface ModelInterface {
+  id: string
+
```

Comment:
Super minor, but as type definitions are already very vertical, could we save some scrolling by removing the empty lines in type definitions?